src/components/Configure/content/fields/useFields.tsx

```diff
@@ -0,0 +1,18 @@
+import { useProject } from '../../../../context/ProjectContext';
+import { useSelectedObjectName } from '../../ObjectManagementNav';
+import { useConfigureState } from '../../state/ConfigurationStateProvider';
+import { getConfigureState } from '../../state/utils';
+
+export const useFields = () => {
```

Comment:
I wonder if `useConfigure` might be a more suitable function name?

Reply:
I think that makes sense, but I think it might get confusing with useConfigureState(). I might think of renaming both hooks. the current naming convention hopefully scopes it to just the fields folder for the time being.